docnado/docnado.py

```diff
@@ -431,7 +431,10 @@ def build_nav_menu(meta_cache):
     This uses the `nav: Assembly>Bench>Part` variable at the top of the Markdown file.
     """
     root = NavItem('root', 0)
-    for path, meta in meta_cache.items():
+    # Pre-sort the nav-items alphabetically by nav-string. This will get overridden with the arange()
+    # function, but this avoids-un arranged items moving round between page refreshes due to Dicts being 
```

Comment:
For readability, what do you think about splitting the sorted out into a separate line and change `i` to `item`. 

Is this also sorting by the *last* item in the nav chain (i.e. `Part` given `Assembly>Bench>Part`) rather than the *first* item (`Assembly` in `Assembly>Bench>Part`).